app_flutter/test/service/mock_cocoon.dart

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:app_flutter/service/fake_cocoon.dart';
+import 'package:mockito/mockito.dart';
+
+/// CocoonService for checking interactions.
+class MockCocoonService extends Mock implements FakeCocoonService {}
```

Comment:
Its fairly common to leave mocks as a single line declaration at the bottom of the test they are used in

Reply:
If this gets used in another file (maybe in another PR), would I move it to its own file?

Reply:
Not generally, since it so small